### DISCUSSION TITLE
Transferring from mleibman

### DISCUSSION THREAD

**Discussion by propkitty** [6pac/SlickGrid#729] [Q&A] [answered]
We recently discovered this new, maintained version of Slickgrid and would like to upgrade to it from the original mleibman Slickgrid. Is there anything that particularly needs to be done or can it be just dropped in and function from the ground up? I can't find any mention of how compatible the old code is with the new or how to translate the old to the new.

Thank you!

**Comment by ghiscoding** [comment, marked as answer]
Apart from the recent [Migration 3.0](https://github.com/6pac/SlickGrid/wiki/Major-version-3.0----Removal-of-jQueryUI-requirement-(replaced-by-SortableJS)) there aren't much more differences. You should simply look at all available [Examples](https://github.com/6pac/SlickGrid/wiki/Examples) to guide you

Note, I'm converting this to a Discussion since it's seems to be more adequate and isn't an Issue per say

**Comment by 6pac** [comment]
@propkitty the new version supports modern versions of jQuery. The dependency on jQueryUI has been removed (including the custom jquery.event.drag.js and jquery.event.drop.js), but in exchange there is now a dependency on SortableJS.
There is a PR in progress currently to remove the dependency on jQuery entirely.

**Reply by propkitty** [reply]
6pac, could not be happier to read this if I tried! I ended up finding this due to the whole jquery.event.drag issue making it a right pain to deal with Slickgrid in webpack. Thank you for the hard work all of you have been doing! I saw on the website that your team is trying to make freeze-columns happen and boy is that something I'm stoked to see work on as our hack-job solutions have not worked properly with 11ish year old code.

**Reply by 6pac** [reply]
@propkitty great to hear! You mainly have @ghiscoding to thank for the new innovations. I'm really lazy updating the website/Github pages, in fact I'll make it a priority over the next week to update them - we've had frozen rows/columns for over a year now.
